docs/DeveloperGuide.adoc

```diff
@@ -111,6 +111,19 @@ image::DeleteSequenceDiagram.png[]
 
 NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
+.Interactions inside the Logic Component for the `enqueue 001A` Command
+image::EnqueueSequenceDiagram.png[]
```

Comment:
Do you want to split the diagram horizontally so that it is easier to read the labels on the diagram?
I feel that the diagram is currently a little hard to read.